staging/src/k8s.io/client-go/tools/clientcmd/overrides.go

```diff
@@ -116,22 +131,23 @@ func (f FlagInfo) BindBoolFlag(flags *pflag.FlagSet, target *bool) FlagInfo {
 }
 
 const (
-	FlagClusterName  = "cluster"
-	FlagAuthInfoName = "user"
-	FlagContext      = "context"
-	FlagNamespace    = "namespace"
-	FlagAPIServer    = "server"
-	FlagAPIVersion   = "api-version"
-	FlagInsecure     = "insecure-skip-tls-verify"
-	FlagCertFile     = "client-certificate"
-	FlagKeyFile      = "client-key"
-	FlagCAFile       = "certificate-authority"
-	FlagEmbedCerts   = "embed-certs"
-	FlagBearerToken  = "token"
-	FlagImpersonate  = "as"
-	FlagUsername     = "username"
-	FlagPassword     = "password"
-	FlagTimeout      = "request-timeout"
+	FlagClusterName       = "cluster"
+	FlagAuthInfoName      = "user"
+	FlagContext           = "context"
+	FlagNamespace         = "namespace"
+	FlagAPIServer         = "server"
+	FlagAPIVersion        = "api-version"
+	FlagInsecure          = "insecure-skip-tls-verify"
+	FlagCertFile          = "client-certificate"
+	FlagKeyFile           = "client-key"
+	FlagCAFile            = "certificate-authority"
+	FlagEmbedCerts        = "embed-certs"
+	FlagBearerToken       = "token"
+	FlagImpersonate       = "as"
+	FlagImpersonateGroups = "asgroups"
```

Comment:
`--as-group`, and can be repeated `--as-group=foo --as-group=bar --as-group=baz`?